jobs/e2e_node/crio/templates/base/crio-install-enable-pod-events.yaml

```diff
@@ -0,0 +1,23 @@
+---
+systemd:
+  units:
+    - name: crio-install.service
+      enabled: true
+      contents: |
+        [Unit]
+        Description=Download and install crio binaries and configurations.
+        After=network-online.target
+
+        [Service]
+        Type=oneshot
+        Environment=ENABLE_POD_EVENTS=true
```

Comment:
as a follow up, we could consider using a drop-in systemd unit for this, or maybe adding an env file

Reply:
Indeed, I'll take care of that!